### PR TITLE
ci: deploy erasure coded pool

### PR DESCRIPTION
**Pull request by Yuggupta27**
deploy erasure coded pool while deploying rook ceph cluster in the CI.

Depends-On: #2654
Signed-off-by: Yug Gupta <yuggupta27@gmail.com>

### CODE REVIEW

File: single-node-k8s.sh [hunk 1]
```diff
@@ -110,6 +110,7 @@ function deploy_rook()
 {
     ${GOPATH}/src/github.com/ceph/ceph-csi/scripts/minikube.sh deploy-rook
     ${GOPATH}/src/github.com/ceph/ceph-csi/scripts/minikube.sh create-block-pool
+    ${GOPATH}/src/github.com/ceph/ceph-csi/scripts/minikube.sh create-ec-pool
```
Comment (Madhu-1) [inline single-node-k8s.sh]: `create-ec-pool`  to `create-block-ec-pool` we also have filesystem EC pools lets make the option specific to block

Comment (Yuggupta27) [reply single-node-k8s.sh]: Okay :+1:

File: single-node-k8s.sh [hunk 2]
```diff
@@ -110,6 +110,7 @@ function deploy_rook()
 {
     ${GOPATH}/src/github.com/ceph/ceph-csi/scripts/minikube.sh deploy-rook
     ${GOPATH}/src/github.com/ceph/ceph-csi/scripts/minikube.sh create-block-pool
+    ${GOPATH}/src/github.com/ceph/ceph-csi/scripts/minikube.sh create-block-ec-pool
```
Comment (nixpanic) [inline single-node-k8s.sh:113]: This means that the change in `rook.sh` in the `devel` branch needs to be backported to all active `release-*` branches.

Comment (Madhu-1) [reply single-node-k8s.sh:113]: Yes correct at least to 3.4 branch

Comment (Yuggupta27) [reply single-node-k8s.sh:113]: Should we ask mergify to backport https://github.com/ceph/ceph-csi/pull/2654 to release v3.4?

Comment (nixpanic) [reply single-node-k8s.sh:113]: yes, add the required backport labels

Comment (Yuggupta27) [reply single-node-k8s.sh:113]: Done

**Comment by nixpanic** [comment]
#2661 should get merged before this, right?

**Review by Rakshith-R — APPROVED** [review]

**Review by Madhu-1 — APPROVED** [review]

**Pull request merged**